src/Models/Constants/Constants.ts

```diff
@@ -439,3 +443,38 @@ export const BlobStorageServiceCorsAllowedHeaders = [
     'x-ms-version',
     'x-ms-blob-type'
 ];
+
+export const EnforcedStorageAccountAccessRoleIds: Array<AzureAccessPermissionRoles> = [];
```

Comment:
Personally I would have left these where they are used since they don't seem to be used across multiple places.

Reply:
Not this particular one, but, I am using other consts that I defined there (`RequiredAccessRoleGroupFor...`) across many files including stories, that is why I moved them here